### PR TITLE
v1

### PR DESCRIPTION
**Pull request by k0nG**
Think this is good for v1. Nice to have it on master.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,175 @@
...
+A sprint is not a fixed set of work. It is an estimation of the amount of work thought possible given the amount of time available. 
+
+During a sprint, if work is progressing at a greater pace than originally anticipated, it is possible to include extra stories from the backlog. 
+
+Likewise, if a sprint is more complex than originally thought, it may be that stories will not be completed. These stories will need to be reconsidered on an individual basis - either they are re-added to the backlog, or they can carry over into the following sprint. However, where possible this should be avoided.
+
+#### At the start of each sprint
+We should have a kick-off meeting where we look through the backlog (which should have been prioritised) and pick a set of stories that can be achieved in the time available. 
+
+Developers will discuss the stories that the business/product owner has provided, discuss them in detail, add technical details and potentially split the stories up into smaller logical chunks of work which can be recorded as tasks on Trello. ** Should this be re-worded?  'smaller sub-stories' or similar, rather than 'tasks'? **
```
Comment (thegingerbloke) [inline README.md]: This comment needs amending/removing

File: README.md [hunk 2]
```diff
@@ -1,2 +1,175 @@
...
+
+The exception being when it's something multiple developers are working on together, in this case the prefix should be `feature/`
+
+Any experimental branches that are to live in the repo long-term should have a prefix of `keep/`, to denote useful work that shouldn't be merged into the main project.
+
+#### Pull requests
+
+No work should ever be directly merged into the main branch. All work should be peer-reviewed via a pull request (PR). Developers on the team should aim to keep the number of changes/commits in the PR to a minimum. 
+
+In the main PR description a link to the story should be provided, along with a description of the work carried out. Disucssion within the PR should be kept to specifics covered in the changed/new code within that PR.
```
Comment (thegingerbloke) [inline README.md]: Spelling - _discussion_

File: README.md [hunk 3]
```diff
@@ -1,2 +1,175 @@
...
+#### Email
+Where possible email should be avoided for project-specific conversation. Where email is necessary, ensure everyone relevant is copied in. 
+
+Email should be used for admin, typically where permanent copies of documents are required by all involved parties - e.g. invoicing, agreeing terms of the project/contract, etc.
+
+## Any other business
+
+### Working remotely
+
+Before someone is able to work remotely, there must be agreement from all concerned. If someone is working remotely they are expected to keep to standard office hours, be present on Slack, and contactable via Skype.
```
Comment (jedrichards) [inline README.md]: I find this part overly officious. Maybe something along the following lines would be more appropriate,

```
Depending on client preferences and other factors remote working is not always appropriate for a project, especially during early planning phases. But as long as the infrastructure is in place to support it (IM, video chats, GitHub etc.) we we think working remotely can be a viable and productive way to approach project work.
```

**Comment by jedrichards** [comment]
Made a small suggestion, otherwise :+1:

**Comment by thegingerbloke** [comment]
:+1:

**Pull request merged**